lib/Gocdb_Services/PI/GetDowntime.php

```diff
@@ -863,11 +882,17 @@ private function getXMLNestedNoPage($downtimes)
                 if ($this->renderMultipleEndpoints) {
                     // Slice the service's endpointLocations array and store just each endpointLocation's Id
                     $currentServiceEndpointIDs = array();
+
+                    // No benefit sorting
```

Comment:
No benefit sorting?

Reply:
In general, I'd expect `$service['endpointLocations']` to be unsorted, so would need sorting added via `orderArrById()`. However, each `$serviceEndpoint` is only used to populate `$currentServiceEndpointIDs`, and the order of that doesn't matter, as the entire array is considered: `if (in_array($dtEndpoint['id'], $currentServiceEndpointIDs))`